docs/index.md

```diff
@@ -31,7 +31,7 @@ For more granular packages we recommend reading the documentation.
 - **Telemetry**
     - [Enrich with telemetry information](features/telemetry)
 - **Logging**
-    - [Logging unhandled exceptions](features/logging)
+    - [Logging](features/logging)
```

Comment:
This is duplicate from the header, let's add an entry for both features to improve discoverability

Reply:
Yes.